modules/dnn/src/layers/layers_common.simd.hpp

```diff
@@ -42,6 +42,23 @@
 
 #include "opencv2/core/hal/intrin.hpp"
 
+#ifndef FAST_CONV_PRAM
```

Comment:
since you've added separate fast_convolution.simd.hpp, why not move all the new convolution kernels there? Please, move conv_block and depthWiseBlock there. At once, please, normalize the naming. If you use lowercase names with underscores for conv_block, use the same style for depthwise_block. Or make everything mixed case: convBlock, depthWiseBlock

Reply:
Fixed.